components/ListItem.js

```diff
@@ -1,17 +1,35 @@
-import react from 'react';
+import React from 'react'
 import Truncate from 'react-truncate'
+const { string, number, shape, arrayOf, func } = React.PropTypes
 
 const ListItem = (props) => (
   <li className="list-item">
-  <a className="list-item-link" href={props.html_url}>
-    <h3 className="list-item__title">
-      <Truncate lines={2}>{props.title}</Truncate>
-    </h3>
-    <p className="list-item__description">
-      <Truncate lines={3}>{props.body || 'No description on this issue'}</Truncate>
-    </p>
-  </a>
-</li>
+    <div className="row">
```

Comment:
Add this `div` inside the `<a className="list-item-link" href={props.html_url}>` and remove the class `row` of it